src/main/g8/build.sbt

```diff
@@ -1,6 +1,8 @@
 ThisBuild / organization := "com.example"
 ThisBuild / scalaVersion := "2.13.5"
 
+addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
```

Comment:
I'd add it to the `root` project's settings. This probably does the same, but in a project with explicitly named sub-projects I prefer not to use this kind of syntax (either moving to `root` settings, or making it a `ThisBuid` scoped setting (`ThisBuild / libraryDependencies += compilerPlugin(...)`) would work, with a slight preference on the former)

Reply:
Do you prefer something like
```scala
    "org.typelevel" %% "cats-effect-kernel" % "3.0.0",
    // standard "effect" library (Queues, Console, Random etc.)
    "org.typelevel" %% "cats-effect-std" % "3.0.0",
    // better monadic for compiler plugin as suggested by documentation
    compilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
  )

or a separate ModuleID added to libDeps like `libraryDependencies += compilerPlugin(...)`?

Reply:
First option works well ☺️

Reply:
Solved.